src/WinUI/ArcGIS.WinUI.Viewer/FeedbackPage.xaml.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright 2024 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Microsoft.UI.Xaml.Controls;
+using ArcGIS.Samples.Managers;
+
+namespace ArcGIS
+{
+    public sealed partial class FeedbackPage : UserControl
+    {
+
+        public FeedbackPage()
+        {
+            this.InitializeComponent();
+        }
+
+        private void BugReportButton_Click(object sender, Microsoft.UI.Xaml.RoutedEventArgs e)
+        {
+            string link =
+                "https://github.com/Esri/arcgis-maps-sdk-dotnet-samples/issues/new?assignees=&labels=Type%3A+Bug&projects=&template=bug_report.yml&title=%5BBug%5D";
+
+            if (SampleManager.Current.SelectedSample != null)
+            {
+                link += "+&impacted-samples=" + SampleManager.Current.SelectedSample.FormalName;
+            }
+
+            _ = Windows.System.Launcher.LaunchUriAsync(new System.Uri(link));
```

Comment:
The link is being created correctly with the impacted sample but I'm not seeing the sample name populated in the opened bug/feature.

`https://github.com/Esri/arcgis-maps-sdk-dotnet-samples/issues/new?assignees=&labels=Type%3A+Bug&projects=&template=bug_report.yml&title=%5BBug%5D+&impacted-samples=AddVectorTiledLayerFromCustomStyle`

![image](https://github.com/Esri/arcgis-maps-sdk-dotnet-samples/assets/99325835/81984765-ba22-4155-8dbb-30c88bf683a1)

Reply:
Issue templates were only committed to v.next at time of opening this PR. I've since cherry-picked that commit to main, so issue templates will now populate sample name in an impacted samples textarea